Practica real con tabla/ViewController.swift

```diff
@@ -94,7 +105,7 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
         var ballon:UIImage = UIImage()
 
 
-        if (swiche.on == true ) {
+        if (message.isOutgoing) {
```

Comment:
Instead of using `switch.on`, we use the flag of `message` to check if the message is sent by me.